core/new-gui/src/app/common/service/user/user.service.ts

```diff
@@ -66,8 +67,8 @@ export class UserService {
         .subscribe(() => this.changeUser(undefined));
   }
 
-  public getUser(): User|undefined {
-    return this.currentUser;
+  public getUser(): Observable<User|undefined> {
+    return of(this.currentUser);
```

Comment:
don't we already have another observable below?
```
 public userChanged(): Observable<User|undefined> {
```
If you want the observable to emit undefined in the beginning, you can either add a `startsWith` to the observable, or you can use a BehaviorSubject (this might fit your use case better) https://www.learnrxjs.io/learn-rxjs/subjects/behaviorsubject

Reply:
changed to use `ReplaySubject` which replays one last value.